.travis.yml

```diff
@@ -4,8 +4,8 @@ android:
   components:
     - tools
     - platform-tools
-    - build-tools-25.0.2
-    - android-25
+    - build-tools-27.0.3
```

Comment:
It seems that Travis still needs to install the build-tools, even if you don't have to specify exactly which ones in the files anymore.

Reply:
Which actually makes me a bit nervous, because by removing the specific build-tools version, we're now creating non-deterministic builds - this build won't work a year from now (on Travis at least).

Reply:
AGP Plugin will download the dependencies.	The only thing that we'll need is:

```
before_install:
  # Install SDK license so Android Gradle plugin can install deps.
  - mkdir "$ANDROID_HOME/licenses" || true
  - echo "d56f5187479451eabf01fb78af6dfcb131a6481e" > "$ANDROID_HOME/licenses/android-sdk-license"
```